### PR TITLE
Not using namespaces when triggering exception causes fatal error in Laravel 5

### PR DESCRIPTION
**Pull request by aginev**
In my case I get this bug (in Laravel 5 only) when I'm tring to upload bigger files thene the max allowed upload size in PHP configuration.

### CODE REVIEW

File: src/JildertMiedema/LaravelPlupload/Receiver.php [hunk 1]
```diff
@@ -91,7 +92,7 @@ public function hasChunks()
 
     public function receive($name, Closure $handler)
     {
-        $response = array();
+        $response = [];
```
Comment (jildertmiedema) [inline src/JildertMiedema/LaravelPlupload/Receiver.php]: Why is this needed?

**Comment by jildertmiedema** [comment]
Please don't use php 5.4 array notation, according the composer.json the minimum php version is 5.3. This will be upgraded, but not for the moment.

File: src/JildertMiedema/LaravelPlupload/Receiver.php (hunk 1 above)
Comment (aginev) [inline src/JildertMiedema/LaravelPlupload/Receiver.php]: You are right, not needed at all. I did not rember to change it... probably when I'm importing the namespaces with the build in function in PhpStorm changes this line like that. Should I fix it and submit a new pull request?

Comment (jildertmiedema) [inline src/JildertMiedema/LaravelPlupload/Receiver.php]: if you correct it and the PR will be updated and I wil merge it

**Comment by jildertmiedema** [comment]
Thanks
